scripts/src/internal/jwt.ts

```diff
@@ -0,0 +1,47 @@
+import * as moment from "moment";
+import { readFileSync } from "fs";
+import * as jsonwebtoken from "jsonwebtoken";
+
+import { getConfig } from "./config";
+
+const CONFIG = getConfig();
+
+type KeyMap = Map<string, { algorithm: string; key: Buffer; }> & {
+	random(): { id: string; key: Buffer; algorithm: string; };
+};
+const KEYS = (() => {
+	const map = new Map<string, { algorithm: string; key: Buffer; }>() as KeyMap;
+	map.random = function(this: Map<string, { algorithm: string; key: Buffer; }>) {
+		const entries = Array.from(this.entries()).map(([id, key]) => ({
+			id,
+			key: key.key,
+			algorithm: key.algorithm
+		}));
+
+		return entries[Math.floor(Math.random() * entries.length)];
+	};
+
+	return map;
+})();
+
+export function sign(subject: string, payload: any) {
+	const signWith = KEYS.random();
+
+	payload = Object.assign({
+		typ: "JWT"
+	}, payload);
+
+	return jsonwebtoken.sign(payload, signWith.key, {
+		subject,
+		keyid: signWith.id,
```

Comment:
great I only now found out that `keyid` is a standard! couldn't find it before so I came up with `key_id`